src/main/java/com/liferay/faces/util/logging/LoggerFactory.java

```diff
@@ -79,23 +70,12 @@
 	 */
 	public static final Logger getLogger(String name) {
 
-		Logger logger = null;
-
-		try {
```

Comment:
@stiemannkj1 Again, I think you are correct. However, perhaps the constructor of LoggerLog4JImpl should throw some type of caught exception, so that we would know that we need to return a LoggerDefaultImpl instead.